p/scripts/install.js

```diff
@@ -1,26 +1,45 @@
 // @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-3.0
 'use strict';
 
-function show_password(ev) {
-	const button = ev.currentTarget;
-	const passwordField = document.getElementById(button.getAttribute('data-toggle'));
+let timeoutHide;
+
+function showPW_this(ev) {
+	const id_passwordField = this.getAttribute('data-toggle');
+	if (this.classList.contains('active')) {
+		hidePW(id_passwordField);
+	} else {
+		if (ev.type === 'click' || ev.buttons || ev.key === ' ' || ev.key.toUpperCase() === 'ENTER') {
```

Comment:
The values of `ev.key` here are standardized btw, so it's always `Enter` on all browsers.

https://www.w3.org/TR/uievents-key/#keys-whitespace

Reply:
It is nice to know. I did just copy&paste from `extra.js`. I would keep it as it is right now.
With a refactoring, we could find a better solution as duplicated source code

Reply:
the user could tab from the password input to the show-pw-button and then press `enter` or `space`

Reply:
If that didn't generate a click event, this code would never be reached. ;-) That would only be relevant in combination with a `btn.addEventListener('keydown', showPW_this);` (or key-anything), which to be clear is not necessary because pressing space or enter on a button or a link generates a click event.

Reply:
you are right. Now it is improved. Thanks!